doc/wakaama_features.rst

```diff
@@ -0,0 +1,213 @@
+Wakaama Features
+----------------
+
+This table shows the LwM2M features that are supported by Wakaama.
+
++----------------------------+----------------------------------------+------------------+-------------------------------+
+| Feature Type               | Feature                                | Wakaama supports | Note                          |
++============================+========================================+==================+===============================+
+| Transport                  | DTLS                                   | v_1.0            | by tinydtls                   |
++                            +----------------------------------------+------------------+-------------------------------+
```

Comment:
nit: It seems to work as is, but this seems cleaner to me:

```suggestion
|                            +----------------------------------------+------------------+-------------------------------+
```

However, I have not found a documentation either is the better way.